repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/config/http-client-service.xsd

```diff
@@ -59,25 +58,46 @@
         </xs:annotation>
 
         <xs:sequence>
-            <xs:element name="pool" type="PoolType" minOccurs="1" maxOccurs="unbounded"/>
+            <xs:element name="client" type="HttpClientConfig" minOccurs="1" maxOccurs="unbounded"/>
```

Comment:
So the problem with this change is that everything else refers to `connection-pool-id`s.

Reply:
Ugh, that's true. If we are breaking things for 9 anyway, I wouldn't be overly opposed to renaming those, but I suppose it's only worth doing if we really care about how they are named. I wanted to change it since `pool` is kind of a misnomer given the configuration options, but we have used that term in a bunch of places now.

Reply:
Maybe split the difference and call it `clientPool` or `client-pool`